src/app/components/trade/SupplyLiquidityModal.tsx

```diff
@@ -186,15 +182,15 @@ export default function SupplyLiquidityModal({ isOpen, onClose, parsedAmounts, c
         .Dex.add(
           baseToken.address,
           quoteToken.address,
-          BalancedJs.utils.toLoop(pool?.baseDeposited || new BigNumber(0), selectedPair.baseCurrencyKey),
-          BalancedJs.utils.toLoop(pool?.quoteDeposited || new BigNumber(0), selectedPair.quoteCurrencyKey),
+          baseDeposit?.quotient?.toString() ?? '0',
```

Comment:
Any specific reason to use `quotient` directly?

Reply:
The `BalancedJs.utils.toLoop` function will be deprecated. So I'd like you to not use the function too in the future. We will refactor that in December.